scaffold/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "",
+  "name": "create-carbon-app",
```

Comment:
Think this needs to be blank so the user of create-carbon-app can name their app etc

Reply:
I'll change it to a different name, the linter was complaining when blank.